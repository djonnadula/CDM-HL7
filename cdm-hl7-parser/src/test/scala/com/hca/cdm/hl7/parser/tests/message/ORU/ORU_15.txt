MSH|^~\&|MT_COCQA1A|COCQA1A|DBM||201601141519||ORU^R01|MT_COCQA1A_ORU_262184..LAB.QA1.567|D|2.5||KYA
PID|1||J000423598|J500217|Jones^Jessica^Campbell^^^|UNKNOWN|20031230|F|CAGE^JESSICA^CAMPBELL JONES^^^|U|456 MARVEL STREET^^FRANKLIN^TN^37064||(615)555-6261|(615)555-6261||S|NON|J00021004019|302-30-2015
PV1|1|I|J.PSY^J.101^A|EL|||DR.ATND01^ATTENDING01^PROVIDER^^^^||DRBRI^DOCTOR^BRITTANY^^^^||||||||DR.ADM01^ADMITTING01^PROVIDER^^^^|IN||08|||||||||||||||||||COCQA1A||ADM|||201601141426
OBR|1|L66918^LAB|^LAB|BMP^BASIC METABOLIC PROFILE^L|R||201601141502|||DRBRI|||PNEUMONIA|201601141502||DR.ATND01^ATTENDING01^PROVIDER^^^^||0114:QAX:C00009R||||201601141519|||F
NTE|1||Could not calculate - patient outside of age range
OBX|1|ST|NA^SODIUM^L|1|135|mEq/L|135-145|N||AS|F|201601111518|^^^ML^QA1 MAIN LAB||^
OBX|2|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|3|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|4|ST|K^POTASSIUM^L|1|3.3|mEq/L|3.5-5.0|L||AS|F|201601120840|^^^ML^QA1 MAIN LAB||^
OBX|5|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|6|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|7|ST|CL^CHLORIDE^L|1|96|mEq/L|98-111|L||AS|F|201601111421|^^^ML^QA1 MAIN LAB||^
OBX|8|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|9|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|10|ST|CO2^CARBON DIOXIDE^L|1|22|mEq/L|22-31|N||AS|F|201512161604|^^^ML^QA1 MAIN LAB||^
OBX|11|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|12|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|13|ST|GAP^ANION GAP^L|1|3||3-15|N||AS|F|201512161605|^^^ML^QA1 MAIN LAB||^
OBX|14|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|15|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|16|ST|GLUX^GLUCOSE^L|1|60|MG/DL|60-130|N||AS|F|201408211139|^^^ML^QA1 MAIN LAB||^
OBX|17|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|18|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|19|ST|BUN^BLOOD UREA NITROGEN^L|1|5|mg/dL|5-26|N||AS|F|201408211139|^^^ML^QA1 MAIN LAB||^
OBX|20|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|21|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|22|ST|GFR^GLOMERULAR FILTRATION RATE^L|1|59|mEq/L|59|N||AS|F|201411111534|^^^ML^QA1 MAIN LAB||^
OBX|23|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|24|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|25|ST|CREAT^CREATININE^L|1|0.6|mg/dL|0.6-1.7|N||AS|F|201408211139|^^^ML^QA1 MAIN LAB||^
OBX|26|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|27|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|28|ST|BUN/CREA^BUN/CREATININE RATIO^L|1|26||5-26|N||AS|F|201408211139|^^^ML^QA1 MAIN LAB||^
OBX|29|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|30|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
OBX|31|ST|CA^CALCIUM^L|1|8.4|mg/dL|8.4-10.6|N||AS|F|201601111422|^^^ML^QA1 MAIN LAB||^
OBX|32|CE|^^||^^||||||||^^^^||ML^QA1 MAIN LAB
OBX|33|AD|2545 Park Plaza|Building 3, 1 West|Nashville|TN|37203
