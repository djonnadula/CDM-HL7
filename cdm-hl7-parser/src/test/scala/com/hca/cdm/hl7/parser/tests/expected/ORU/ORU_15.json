{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "MT_COCQA1A_ORU_262184..LAB.QA1.567",
    "medical_record_num": "J000423598",
    "medical_record_urn": "J500217",
    "patient_account_num": "J00021004019"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "MT_COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "DBM",
    "receiving_facility": null,
    "date_time_of_message": "201601141519",
    "security": null,
    "message_type": {
      "message_code": "ORU",
      "trigger_event": "R01"
    },
    "message_control_id": "MT_COCQA1A_ORU_262184..LAB.QA1.567",
    "processing_id": "D",
    "version_id": "2.5",
    "sequence_num": null,
    "continuation_pointer": "KYA"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": "J000423598",
    "alternate_patient_id": "J500217",
    "patient_name": {
      "family_name": "Jones",
      "given_name": "Jessica",
      "second_and_further_given_names_or_initials_thereof": "Campbell",
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "mother_maiden_name": "UNKNOWN",
    "date_time_of_birth": "20031230",
    "administrative_sex": "F",
    "patient_alias": {
      "family_name": "CAGE",
      "given_name": "JESSICA",
      "second_and_further_given_names_or_initials_thereof": "CAMPBELL JONES",
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "race": "U",
    "patient_address": {
      "street_address": "456 MARVEL STREET",
      "other_designation": null,
      "city": "FRANKLIN",
      "state_or_province": "TN",
      "zip_or_postal_code": "37064"
    },
    "county_code": null,
    "phone_num_home": "(615)555-6261",
    "phone_num_business": "(615)555-6261",
    "primary_language": null,
    "marital_status": "S",
    "religion": "NON",
    "patient_account_num": "J00021004019",
    "ssn_num_patient": "302-30-2015"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "J.PSY",
      "room": "J.101",
      "bed": "A"
    },
    "admission_type": "EL",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "DR.ATND01",
      "family_name": "ATTENDING01",
      "given_name": "PROVIDER",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "referring_doctor": null,
    "primary_care_doctor": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": null,
    "mode_of_arrival": null,
    "vip_indicator": null,
    "admitting_doctor": {
      "person_id": "DR.ADM01",
      "family_name": "ADMITTING01",
      "given_name": "PROVIDER",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "patient_type": "IN",
    "visit_num": null,
    "financial_class": "08",
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": "COCQA1A",
    "reason_for_visit": null,
    "account_status": "ADM",
    "pending_location": null,
    "prior_temporary_location": null,
    "admit_date_time": "201601141426"
  },
  "0004.OBR": {
    "set_id": "1",
    "placer_order_num": {
      "entity_id": "L66918",
      "namespace_id": "LAB"
    },
    "filler_order_num": {
      "entity_id": null,
      "namespace_id": "LAB"
    },
    "universal_service_id": {
      "id": "BMP",
      "text": "BASIC METABOLIC PROFILE",
      "name_of_coding_system": "L"
    },
    "priority": "R",
    "requested_date_time": null,
    "obsv_date_time_num": "201601141502",
    "obsv_end_date_time_num": null,
    "collection_volume": null,
    "collector_id": "DRBRI",
    "specimen_action_code": null,
    "danger_code": null,
    "relevant_clinical_information": "PNEUMONIA",
    "specimen_received_date_time": "201601141502",
    "specimen_source": null,
    "ordering_provider": {
      "person_id": "DR.ATND01",
      "family_name": "ATTENDING01",
      "given_name": "PROVIDER",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "order_callback_phone_num": null,
    "placer_field_1": "0114:QAX:C00009R",
    "placer_field_2": null,
    "filler_field_1": null,
    "filler_field_2": null,
    "results_rpt_status_chng_date_time": "201601141519",
    "charge_to_practice": null,
    "diagnostic_service_section_id": null,
    "result_status": "F"
  },
  "0005.NTE": {
    "set_id": "1",
    "source_of_comment": null,
    "comment": "Could not calculate - patient outside of age range"
  },
  "0006.OBX": {
    "set_id": "1",
    "value_type": "ST",
    "obsv_id": {
      "id": "NA",
      "text": "SODIUM",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "135",
    "units": "mEq/L",
    "references_range": "135-145",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201601111518",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0007.OBX": {
    "set_id": "2",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0008.OBX": {
    "set_id": "3",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0009.OBX": {
    "set_id": "4",
    "value_type": "ST",
    "obsv_id": {
      "id": "K",
      "text": "POTASSIUM",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "3.3",
    "units": "mEq/L",
    "references_range": "3.5-5.0",
    "interpretation_codes": "L",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201601120840",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0010.OBX": {
    "set_id": "5",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0011.OBX": {
    "set_id": "6",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0012.OBX": {
    "set_id": "7",
    "value_type": "ST",
    "obsv_id": {
      "id": "CL",
      "text": "CHLORIDE",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "96",
    "units": "mEq/L",
    "references_range": "98-111",
    "interpretation_codes": "L",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201601111421",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0013.OBX": {
    "set_id": "8",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0014.OBX": {
    "set_id": "9",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0015.OBX": {
    "set_id": "10",
    "value_type": "ST",
    "obsv_id": {
      "id": "CO2",
      "text": "CARBON DIOXIDE",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "22",
    "units": "mEq/L",
    "references_range": "22-31",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201512161604",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0016.OBX": {
    "set_id": "11",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0017.OBX": {
    "set_id": "12",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0018.OBX": {
    "set_id": "13",
    "value_type": "ST",
    "obsv_id": {
      "id": "GAP",
      "text": "ANION GAP",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "3",
    "units": null,
    "references_range": "3-15",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201512161605",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0019.OBX": {
    "set_id": "14",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0020.OBX": {
    "set_id": "15",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0021.OBX": {
    "set_id": "16",
    "value_type": "ST",
    "obsv_id": {
      "id": "GLUX",
      "text": "GLUCOSE",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "60",
    "units": "MG/DL",
    "references_range": "60-130",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201408211139",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0022.OBX": {
    "set_id": "17",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0023.OBX": {
    "set_id": "18",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0024.OBX": {
    "set_id": "19",
    "value_type": "ST",
    "obsv_id": {
      "id": "BUN",
      "text": "BLOOD UREA NITROGEN",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "5",
    "units": "mg/dL",
    "references_range": "5-26",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201408211139",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0025.OBX": {
    "set_id": "20",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0026.OBX": {
    "set_id": "21",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0027.OBX": {
    "set_id": "22",
    "value_type": "ST",
    "obsv_id": {
      "id": "GFR",
      "text": "GLOMERULAR FILTRATION RATE",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "59",
    "units": "mEq/L",
    "references_range": "59",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201411111534",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0028.OBX": {
    "set_id": "23",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0029.OBX": {
    "set_id": "24",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0030.OBX": {
    "set_id": "25",
    "value_type": "ST",
    "obsv_id": {
      "id": "CREAT",
      "text": "CREATININE",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "0.6",
    "units": "mg/dL",
    "references_range": "0.6-1.7",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201408211139",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0031.OBX": {
    "set_id": "26",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0032.OBX": {
    "set_id": "27",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0033.OBX": {
    "set_id": "28",
    "value_type": "ST",
    "obsv_id": {
      "id": "BUN/CREA",
      "text": "BUN/CREATININE RATIO",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "26",
    "units": null,
    "references_range": "5-26",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201408211139",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0034.OBX": {
    "set_id": "29",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0035.OBX": {
    "set_id": "30",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  },
  "0036.OBX": {
    "set_id": "31",
    "value_type": "ST",
    "obsv_id": {
      "id": "CA",
      "text": "CALCIUM",
      "name_of_coding_system": "L"
    },
    "obsv_sub_id": "1",
    "obsv_value": "8.4",
    "units": "mg/dL",
    "references_range": "8.4-10.6",
    "interpretation_codes": "N",
    "probability": null,
    "nature_of_abnormal_test": "AS",
    "obsv_result_status": "F",
    "effective_date_of_reference_range": "201601111422",
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": "ML",
      "alternate_text": "QA1 MAIN LAB"
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": null,
      "text": null
    }
  },
  "0037.OBX": {
    "set_id": "32",
    "value_type": "CE",
    "obsv_id": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": null,
      "name_of_coding_system": null
    },
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": null,
    "effective_date_of_reference_range": null,
    "user_defined_access_checks": {
      "id": null,
      "text": null,
      "name_of_coding_system": null,
      "alternate_id": null,
      "alternate_text": null
    },
    "date_time_of_the_obsv": null,
    "producer_id": {
      "id": "ML",
      "text": "QA1 MAIN LAB"
    }
  },
  "0038.OBX": {
    "set_id": "33",
    "value_type": "AD",
    "obsv_id": "2545 Park Plaza",
    "obsv_sub_id": "Building 3, 1 West",
    "obsv_value": "Nashville",
    "units": "TN",
    "references_range": "37203"
  }
}